code-challenges.test.js

```diff
@@ -12,21 +14,58 @@
 
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
-// a) Create a test with expect statements for each of the variables provided.
+//  a) Create a test with expect statements for each of the variables provided.
+
+describe("greaterThanTwo", () => {
+  it("returns an array that length containing the numbers of the Fibonacci sequence", () => {
+    expect(greaterThanTwo(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+    expect(greaterThanTwo(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+  })
+})
+
+//Good Fail.
+// // ReferenceError: greaterThanTwo is not defined
 
 const fibLength1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
 const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
-// b) Create the function that makes the test pass.
+// // b) Create the function that makes the test pass.
+
+// // PSUEDO CODE:
+// // Create a function named greaterThanTwo.
+// // Create two variables, one that is equal to 0, and one that is equal to 1 as this is what the Fibonacci sequence starts with.
+// // Create the starting sequence (n2) at 1.
+// // Now the for loop is starting at 2 and we must build on it from there meaning the previous values will be added and once they are added we will get the new number.
+
+const greaterThanTwo = (num) => {
+  let n1 = [1, 1]
+  let n2 = [1]
+  for (let i = 2; i < num; i++) {
+    n1[i] = n1[i - 2] + n1[i - 1]
+  }
+  return n1
+}
+console.log(greaterThanTwo(10))
+
```

Comment:
Great work Zack!  As a challenge, try refactoring this with higher order functions.  You're doing great!